src/prometheus_alert_rules/disk.rules

```diff
@@ -29,17 +29,6 @@ groups:
         Host filesystem '{{ $labels.mountpoint }}' is using {{ $value | printf "%.0f" }}% of the total space.
           VALUE = {{ $value }}
           LABELS = {{ $labels }}
-  - alert: HostReadonlyFilesystem
```

Comment:
Good comment from @axinojolais:

> I wouldn’t really care if a squashfs or loop filesystem is read-only, but I would for, say, ext4 because it generally is a sign that a disk error happened. More generally, I think an alert if a filesystem that was read-write suddenly becomes read-only would be helpful. Would there be a way to get that ?